src/components/spinner/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import SC from './styled';
+
+const Spinner: React.FC = () => (
+  <SC.Spinner>
+    <SC.SpinnerImage />
+  </SC.Spinner>
+);
```

Comment:
Om du følger forslaget under kan du bare gjøre:
```suggestion
const Spinner: React.FC = () => <SC.Spinner/>
```